sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/protocol/datalake_rest_client.hpp

```diff
@@ -2367,12 +2367,12 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
               ContentLength; // Required for "Append Data" and "Flush Data".  Must be 0 for "Flush
                              // Data".  Must be the length of the request content in bytes for
                              // "Append Data".
-          Azure::Core::Nullable<ContentHash>
+          Azure::Core::Nullable<Storage::ContentHash>
               TransactionalContentMd5; // Specify the transactional md5 for the body, to be
                                        // validated by the service.
-          Azure::Core::Nullable<ContentHash>
-              ContentCrc64; // Specify the transactional crc64 for the body, to be validated by the
-                            // service.
+          Azure::Core::Nullable<Storage::ContentHash>
+              TransactionalContentCrc64; // Specify the transactional crc64 for the body, to be
+                                         // validated by the service.
```

Comment:
Why it's not named as `TransactionalContentHash`?

Reply:
Customers don't have access to this.

Reply:
Well, customers having no access to it doesn't justify that we can randomly name it.